pkg/apis/metrics/types.go

```diff
@@ -27,3 +30,26 @@ type RawNode struct {
 type RawPod struct {
 	unversioned.TypeMeta `json:",inline"`
 }
+
+// Node-level metrics.
+type Node struct {
+	unversioned.TypeMeta `json:",inline"`
+	api.ObjectMeta       `json:"metadata,omitempty"`
+	Metrics              Metrics `json:"metrics,omitempty"`
+}
+
+// Pod-level metrics.
+type Pod struct {
+	unversioned.TypeMeta `json:",inline"`
+	api.ObjectMeta       `json:"metadata,omitempty"`
```

Comment:
I don't think Pod/Node should have `ObjectMeta`, since they are synthetic objects (a lot of the fields don't apply). In previous discussions, we decided that `ListMeta` or similar metadata type would be a better fit, paired with a `NonLocalObjectReference` (a.k.a. Name + Namespace)